brave/brave-core
# Add statement of accounts changes for Brave Rewards NTP widget

Resolves https://github.com/brave/brave-browser/issues/12880

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:

- Confirm "estimated pending rewards" are correctly shown after viewing ads
- Confirm "next payment date" is correctly shown after viewing ads
- Confirm "ads received this month" are correctly shown after viewing ads
- Confirm "estimated pending rewards" are correctly shown after restarting the browser
- Confirm "next payment date" is correctly shown after restarting the browser
- Confirm "ads received this month" are correctly shown after restarting the browser
